service-api/docker/web/Dockerfile

```diff
@@ -1,11 +1,19 @@
 FROM nginx:stable-alpine
 
+RUN chown -R nginx:nginx /var/cache/nginx && \
+    chown -R nginx:nginx /var/log/nginx && \
+    chown -R nginx:nginx /etc/nginx/conf.d && \
+    touch /run/nginx.pid && \
+    chown -R nginx:nginx /run/nginx.pid && \
+    chmod -R 777 /etc/nginx/conf.d
```

Comment:
```suggestion
    chmod o+rwx /etc/nginx/conf.d
```

Reply:
the above change has been committed and works so far, can you re-review please @nickdavis2001